### ISSUE TITLE
Govt-wide accessibility assessment, pt 2

### ISSUE THREAD

**Issue by eelzi-ONRR** [DOI-ONRR/nrrd#3472]
Finish questions I can answers (reviewed 90 questions last sprint, answered what I could, have 13 more questions to go), assign questions I cannot answer to others in ONRR

**Comment by eelzi-ONRR** [comment]
Answered questions relevant to ODDD, except for top pages and document testing  - issues #3185, #3184, #3490

Split up questions and sent to Christa M., Chris M., Sheila P., Deb & Heidi, Joe M., Bill B., and Roland R. Met with Christina M. for help on a few questions. 

Answers due to me on July 19 so I can review and compile for July 31 due deadline.
